Tests/scripts/wait_until_server_ready.py

```diff
@@ -58,6 +59,17 @@ def download_cloud_init_logs_from_server(ip: str) -> None:
         logging.exception(f'Could not download cloud-init file from server {ip}.')
 
 
+def docker_login(ip: str) -> None:
+    """
+    Login-in to docker on the server to avoid docker rate limit quota violation
+    Args:
+        ip: The ip of the server that should be logged in
+    """
+    docker_username = os.environ.get('DOCKERHUB_USER')
+    docker_password = os.environ.get('DOCKERHUB_PASSWORD')
+    check_output(f'echo {docker_password} | ssh -o StrictHostKeyChecking=no ec2-user@{ip} '
```

Comment:
This doesn't work. The command will simply `echo` everything (including the `|`). You should remove the echo and just pass: `input=docker_password` as an arg.